docker/lib/dependabot/docker/file_updater.rb

```diff
@@ -158,7 +158,7 @@ def update_helm(file, content)
         old_tags.each do |old_tag|
           old_tag_regex = /^\s+(?:-\s)?(?:tag|version):\s+["']?#{old_tag}["']?(?=\s|$)/
           modified_content = modified_content.gsub(old_tag_regex) do |old_img_tag|
-            old_img_tag.gsub(old_tag.to_s, new_yaml_tag(file).to_s)
+            old_img_tag.gsub(old_tag.to_s, new_helm_tag(file).to_s)
```

Comment:
The old method is dead now?

Reply:
Correct, that method is no longer used, I'll remove it shortly.